sympy/series/limitseq.py

```diff
@@ -197,7 +197,7 @@ def limit_seq(expr, n=None, trials=5):
         elif not free:
             return expr
         else:
-            raise ValueError("expr %s has more than one variables. Please"
+            raise ValueError("expr %s has more than one variables. Please "
```

Comment:
This will cause "Pleasespecify" to be printed with no space between the words. It would be better to leave that space there and delete the "s" from "variables."

Reply:
@smichr "Pleasespecify" is the current behavior, this PR _adds_ a space. 

I agree that "has more than one variable" would be more grammatical; in fact I proposed this change in #14198 (along with changing "expr" to "Expression" and some math stuff).

Reply:
Thanking you for pointing that out. I have made that change as well.